src/tests/errors/changelog.errors.test.js

```diff
@@ -4,10 +4,10 @@ import {
   InvalidChangelogHeadingError,
   EntryTooLongError,
   InvalidPrefixError,
-  CategoryWithSkipOptionError,
+  InvalidAdditionalPrefixWithSkipEntryOptionError,
```

Comment:
Where are you using this?

Reply:
So, I made general updates to error class names, which were also in the unit test files. 

But all the tests are old and definitely broken. The ones you see now are the ones I started implementing in this reusable action. But to speed up development, I have to drop the TDD approach. The next step - after finishing the second part of the manual approach (validation of changesets) - will be to start working on the unit tests and more refactoring to make the code more robust.

Reply:
Ah, no worries. Was just checking :) Makes sense